index.html

```diff
@@ -2092,39 +2143,39 @@ <h3>Indirect Communication</h3>
                     <img
                         src="images/arch-high-level-application-proxy-device-alt2.png" />
                     <figcaption>High level architecture
-                        with proxy</figcaption>
+                        with Intermediary</figcaption>
                 </figure>
 
                 <p>
-                    A proxy combines <a>Exposed Thing</a> and <a>Consumed Thing</a>
+                    An <a>Intermediary</a> combines <a>Exposed Thing</a> and <a>Consumed Thing</a>
                     functionality, and relays messages that are exchanged
-                    between a <a>WoT Client</a> and a <a>WoT Server</a>. In a proxy,
+                    between a <a>Consumer</a> and a <a>Thing</a>. In an <a>Intermediary</a>,
                     a <a>Consumed Thing</a> creates a <em>shadow thing
```

Comment:
"shadow thing" seems to need a clear definition.
Maybe we can just talk about "shadow of the Exposed Thing"?
Maybe proxy object or instance might be better? Or "stub"?